public/provisioning.php

```diff
@@ -65,6 +65,9 @@
         $scope_data['provisioning_url_host'] = gethostname();
     }
 
+    // Add user agent
+    $scope_data['user-agent'] = $_SERVER['HTTP_USER_AGENT'];
```

Comment:
The minus "-" sign is often not allowed in variable names

```suggestion
    $scope_data['user_agent'] = $_SERVER['HTTP_USER_AGENT'];
```